intelmq/etc/harmonization.conf

```diff
@@ -197,6 +197,10 @@
             "regex": "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[0-9a-z]{12}$",
             "type": "LowercaseString"
         },
+        "output": {
```

Comment:
Well, I used the name we were talking about at Prague. If you feel 'transformed' is more fitting, I can refactor. However I took this as a temporary solution until next version of IntelMQ anyway - as I understand it, there are plans of rearchitecting configuration/contracts, adding transformation class of bots, extending harmonization (which would allow describing foreign formats, like Idea).